LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -5607,5 +5610,328 @@ LocLongDescription="Free <Ability:WeaponName/> attacks on any enemies that enter
+
+[LW_ScrapMetal X2AbilityTemplate]
+LocFriendlyName="Scrap Metal"
+LocFlyOverText="Scrap Metal: Recovered ammo"
+LocLongDescription="Kills with your Primary restore one Sawed-Off Shotgun charge."
+LocHelpText="Kills with your Primary restore one Sawed-Off Shotgun charge."
+LocPromotionPopupText="<Bullet/> Kills with your Primary restore one Sawed-Off Shotgun charge."
+
+[LW_ScrapMetalTrigger X2AbilityTemplate]
+LocFriendlyName="Scrap Metal: Recovered ammo"
+
+[LW_Brutality X2AbilityTemplate]
+LocFriendlyName="Brutality"
+LocLongDescription="Kills on organic enemies with your Sawed-Off Shotgun have a <Ability:BRUTALITY_PANIC_CHANCE/> chance to panic other organic enemies within <Ability:BRUTALITY_TILE_RADIUS/> tiles."
+LocHelpText="Kills on organic enemies with your Sawed-Off Shotgun have a <Ability:BRUTALITY_PANIC_CHANCE/> chance to panic other organic enemies within <Ability:BRUTALITY_TILE_RADIUS/> tiles."
+LocFlyOverText="Brutality"
+LocPromotionPopupText="<Bullet/> Use Brutality as a crowd control tool to allow the Marauder to move forward more aggressively and combine it with the Ruthless ability to increase your killing power.<br/>"
+
+[LW_Ruthless X2AbilityTemplate]
+LocFriendlyName="Ruthless"
+LocLongDescription="Killing a panicked or disoriented enemy with your Sawed-Off Shotgun refunds one action point."
+LocHelpText="Killing a panicked or disoriented enemy with your Sawed-Off Shotgun refunds one action point."
+LocFlyOverText="Ruthless"
+LocPromotionPopupText="<Bullet/> By cleverly combining the effects of Brutality and Ruthless, you can kill multiple enemies per turn with your Sawed-Off Shotgun.<br/><Bullet/> There is no limit to the number of activations per turn.<br/>"
+
+[LW_ZoneOfControl X2AbilityTemplate]
+LocFriendlyName = "Zone of Control"
+LocHelpText = "All enemies within the <Ability:ClassName/>'s <Ability:CQB_DOMINANCE_RADIUS/> tile radius suffer penalties of <Ability:ZONE_CONTROL_AIM_PENALTY/> Aim and <Ability:ZONE_CONTROL_MOBILITY_PENALTY/> Mobility."
+LocLongDescription = "All enemies within the <Ability:ClassName/>'s <Ability:CQB_DOMINANCE_RADIUS/> tile radius suffer penalties to Aim and Mobility."
+LocPromotionPopupText = "<Bullet/> All enemies within the <Ability:ClassName/>'s <Ability:CQB_DOMINANCE_RADIUS/> tile radius suffer penalties of <Ability:ZONE_CONTROL_AIM_PENALTY/> Aim and <Ability:ZONE_CONTROL_MOBILITY_PENALTY/ Mobility.<br/>"
+
+[LW_ApexPredator X2AbilityTemplate]
+LocFriendlyName = "Apex Predator"
+LocFlyOverText = "Apex Predator"
+LocHelpText = "Critical Hits have a chance to panic the target and nearby enemies."
+LocLongDescription = "Critical Hits have a chance to panic the target and nearby enemies."
+LocPromotionPopupText = "<Bullet/>"Critical Hits have a chance to panic the target and nearby enemies.<br/>"
```

Comment:
```suggestion
LocHelpText = "Critical hits have a chance to panic the target and nearby enemies."
LocLongDescription = "Critical hits have a chance to panic the target and nearby enemies."
LocPromotionPopupText = "<Bullet/> Critical hits have a chance to panic the target and nearby enemies.<br/>"
```

I would ideally like to have an extra line that states the approximate chance to panic a unit with a given Will. For example, is it a 50% chance to panic a unit with 100 Will? I can't remember how that darn algorithm works.